src/engine/strat_engine/mdv.rs

```diff
@@ -63,35 +108,48 @@ impl MetadataVol {
             }
         }
 
-        if let Err(err) = mount(Some(&dev.devnode()),
-                                &mount_pt,
-                                Some("xfs"),
-                                MsFlags::empty(),
-                                None as Option<&str>) {
-            match err {
-                nix::Error::Sys(nix::Errno::EBUSY) => {
-                    // EBUSY is the error returned in this context when the
-                    // device is already mounted at the specified mountpoint.
+        let mut mdv = MetadataVol {
+            dev,
+            mount_pt,
+            is_mounted: false,
```

Comment:
I know we're using this in the code, but it's not clear to me that we're using it for anything useful.

Reply:
ok, removed..